widget/settings/pages/config/bar/index.ts

```diff
@@ -73,13 +73,14 @@ export const BarSettings = () => {
                 Header('Media'),
                 Option({ opt: options.theme.bar.buttons.media.spacing, title: 'Inner Spacing', subtitle: 'Spacing between the icon and the label inside the buttons.', type: 'string' }),
                 Option({ opt: options.bar.media.show_artist, title: 'Show Track Artist', type: 'boolean' }),
-                Option({ opt: options.bar.media.truncation, title: 'Truncate Media Label', type: 'boolean' }),
-                Option({ opt: options.bar.media.truncation_size, title: 'Truncation Size', type: 'number', min: 10 }),
+                Option({ opt: options.bar.media.show_label, title: 'Toggle Media Label', type: 'boolean' }),
+                Option({ opt: options.bar.media.truncation, title: 'Truncate Media Label', subtitle: 'Only applicable if Toggle Media Label is enabled', type: 'boolean' }),
+                Option({ opt: options.bar.media.truncation_size, title: 'Truncation Size', subtitle: 'Only applicable if Toggle Media Label is enabled', type: 'number', min: 0 }),
```

Comment:
```suggestion
                Option({ opt: options.bar.media.truncation_size, title: 'Truncation Size', subtitle: 'Only applicable if Toggle Media Label is enabled', type: 'number', min: 10 }),
```
Let's keep this at 10 since now we have a toggle.